### DISCUSSION TITLE
How to use the function of Automatic Social Images？

### DISCUSSION THREAD

**Discussion by JIeJaitt** [transitive-bullshit/nextjs-notion-starter-kit#425] [Q&A]
I can't use the function of Automatic Social Images normally, how can I use this function? My notion blog can't see the automatically generated article header image

**Comment by JIeJaitt** [comment]
<img width="860" alt="image" src="https://user-images.githubusercontent.com/77219045/208232018-3bdc5490-0057-439e-8d4f-febe6b32f3b5.png">

**Reply by Bandos1000** [reply]
Hey JoeJaitt, which aspect are you talking about? The article header image (covers?) or social media icons on the top right?
